sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -106,7 +107,7 @@ private[this] object JsonPathParser extends RegexParsers {
   }
 }
 
-private[this] object SharedFactory {
+private[expressions] object SharedFactory {
```

Comment:
Currently, we have to do this because the expressions `GetJsonObject` and `JsonTuple` share this class in the file `jsonExpressions.scala`. After this PR, I will move `GetJsonObjectEvaluator` to `JsonExpressionEvalUtils.scala` and restore its access permissions to `private[this]`